### ISSUE TITLE
CharTests failing on Windows after upgrading to rc3-23722

### ISSUE THREAD

**Issue by stephentoub** [dotnet/corefx#5645]
http://dotnet-ci.cloudapp.net/job/dotnet_corefx/job/windows_nt_debug_prtest/1339/consoleFull
```
17:59:44   Discovering: System.Runtime.Tests
17:59:45   Discovered:  System.Runtime.Tests
17:59:45   Starting:    System.Runtime.Tests
17:59:46      CharTests.TestIsLetterStrInt [FAIL]
17:59:46         Assert.False() Failure
17:59:46         Expected: False
17:59:46         Actual:   True
17:59:46         Stack Trace:
17:59:46            d:\j\workspace\dotnet_corefx\windows_nt_debug_prtest\src\System.Runtime\tests\System\Char.cs(294,0): at CharTests.TestIsLetterStrInt()
17:59:46      CharTests.TestIsLower [FAIL]
17:59:46         Assert.False() Failure
17:59:46         Expected: False
17:59:46         Actual:   True
17:59:46         Stack Trace:
17:59:46            d:\j\workspace\dotnet_corefx\windows_nt_debug_prtest\src\System.Runtime\tests\System\Char.cs(359,0): at CharTests.TestIsLower()
17:59:46      CharTests.TestIsNumber [FAIL]
17:59:46         Assert.False() Failure
17:59:46         Expected: False
17:59:46         Actual:   True
17:59:46         Stack Trace:
17:59:46            d:\j\workspace\dotnet_corefx\windows_nt_debug_prtest\src\System.Runtime\tests\System\Char.cs(389,0): at CharTests.TestIsNumber()
17:59:46      CharTests.TestIsLetter [FAIL]
17:59:46         Assert.False() Failure
17:59:46         Expected: False
17:59:46         Actual:   True
17:59:46         Stack Trace:
17:59:46            d:\j\workspace\dotnet_corefx\windows_nt_debug_prtest\src\System.Runtime\tests\System\Char.cs(275,0): at CharTests.TestIsLetter()
17:59:46      CharTests.TestIsDigit [FAIL]
17:59:46         Assert.False() Failure
17:59:46         Expected: False
17:59:46         Actual:   True
17:59:46         Stack Trace:
17:59:46            d:\j\workspace\dotnet_corefx\windows_nt_debug_prtest\src\System.Runtime\tests\System\Char.cs(239,0): at CharTests.TestIsDigit()
17:59:46      CharTests.TestIsNumberStrInt [FAIL]
17:59:46         Assert.False() Failure
17:59:46         Expected: False
17:59:46         Actual:   True
17:59:46         Stack Trace:
17:59:46            d:\j\workspace\dotnet_corefx\windows_nt_debug_prtest\src\System.Runtime\tests\System\Char.cs(402,0): at CharTests.TestIsNumberStrInt()
17:59:46      CharTests.TestIsLetterOrDigit [FAIL]
17:59:46         Assert.False() Failure
17:59:46         Expected: False
17:59:46         Actual:   True
17:59:46         Stack Trace:
17:59:46            d:\j\workspace\dotnet_corefx\windows_nt_debug_prtest\src\System.Runtime\tests\System\Char.cs(321,0): at CharTests.TestIsLetterOrDigit()
17:59:46      CharTests.TestIsDigitStrInt [FAIL]
17:59:46         Assert.False() Failure
17:59:46         Expected: False
17:59:46         Actual:   True
17:59:46         Stack Trace:
17:59:46            d:\j\workspace\dotnet_corefx\windows_nt_debug_prtest\src\System.Runtime\tests\System\Char.cs(250,0): at CharTests.TestIsDigitStrInt()
17:59:46      CharTests.TestIsLowerStrInt [FAIL]
17:59:46         Assert.False() Failure
17:59:46         Expected: False
17:59:46         Actual:   True
17:59:46         Stack Trace:
17:59:46            d:\j\workspace\dotnet_corefx\windows_nt_debug_prtest\src\System.Runtime\tests\System\Char.cs(370,0): at CharTests.TestIsLowerStrInt()
17:59:46      CharTests.TestIsLetterOrDigitStrInt [FAIL]
17:59:46         Assert.False() Failure
17:59:46         Expected: False
17:59:46         Actual:   True
17:59:46         Stack Trace:
17:59:46            d:\j\workspace\dotnet_corefx\windows_nt_debug_prtest\src\System.Runtime\tests\System\Char.cs(342,0): at CharTests.TestIsLetterOrDigitStrInt()
```

**Comment by ellismg** [comment]
Thanks.  To remind myself, it's troubling that the updates didn't cause breaks on OSX and Linux, so I will need to look into that as well.

**Comment by weshaggard** [comment]
@tarekgh just fixed similar tests internally, as I think these failures are a result of the update to Unicode 8.

**Comment by tarekgh** [comment]
@ellismg, I think Wes is right. I can help fixing it. but I think it should fail on OSX and Linux too. please let me know if you want me take a look.

**Comment by tarekgh** [comment]
The fix I did in the TFS is changeset: 1568080 just in case you'll do the same here.

**Comment by tarekgh** [comment]
This is fixed by the linked PR's
